internal/DateTime/DateTime.module.less

```diff
@@ -16,3 +17,31 @@
 		}
 	}
 }
+
+.datePicker, .timePicker {
+	.day,
+	.month,
+	.year,
+	.hourPicker,
+	.minutePicker,
+	.meridiemPicker {
+		position: relative;
+
+		&:nth-child(n+2)::before {
+			content: '';
+			position: absolute;
+			top: 50%;
+			transform: translateY(-50%);
+		}
+
+		.applySkins({
+			&:nth-child(n+2)::before {
+				width: @agate-datetimepicker-divider-width;
+				height: @agate-datetimepicker-divider-height;
+				opacity: @agate-datetimepicker-divider-opacity;
+				background-color: @agate-datetimepicker-divider-bg-color;
+				right: calc(100% - @agate-datetimepicker-divider-width);
+			}
```

Comment:
What do you think about just merging these two blocks, for simplicity? Just do everything inside the `.applySkins` mixin.